src/components/PaginationButtons/PaginationButton.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import './paginationButton.css'
+
+const NavButtons = ({currentPage, allPages, nextPage, prevPage, firstPage, lastPage}) => {
```

Comment:
Try programming the `nextPage`, `prevPage`, `firstPage` and `lastPage` functions inside of this component instead of passing them in via props. You will need to pass an `updatePage`, along with some other props, from your Deals and Home pages.